### ISSUE TITLE
Attempt to index a nil value (field '?') and QBCore.Shared errors

### ISSUE THREAD

**Issue by rpxyz** [jimathy/jim-mining#28]
Hey mate, just came across your script, thank you very much, first of all, it's great and especially for FREE. Though not a game-breaking bug I have some feedback for you. The first one is (Attempt to index a nil value (field '?').

And the 2nd one is an error about QBCore.Shared.Items. (imgs attached) 

![hhhh](https://user-images.githubusercontent.com/2183651/193831000-f8364a36-5829-47e5-8907-583700adf5a6.png)
![Screenshot_1](https://user-images.githubusercontent.com/2183651/193831013-8c6363a0-ce28-4f18-8f1c-14e6b15586b6.png)

**Comment by rpxyz** [comment]
Nope, that is not the problem mate, I can see ALL the items in the game, I can even mine, sell, etc etc... it's all working, BUT those errors are being spammed upon starting the server up. Like I said it isn't game-breaking, just errors in the log. On the CFX Forums someone else is reporting the same problem, it seems to be something with the crafting side. 

Crafting your items doesn't work either, such as making jewelry etc, other than that it's all good.

As you can see:

![Screenshot_1](https://user-images.githubusercontent.com/2183651/193871675-dbf7a336-beb5-4907-bd9d-529656246c92.png)

**Comment by xViperAG** [comment]
Is there a comma next to whatever is above the ```-- Jim-Mining``` ?

**Comment by rpxyz** [comment]
Yes, mate there is, like I said, from experience, this is something in the code/crafting area?!?

**Comment by jimathy** [comment]
Ok, so it really shouldn't be spamming them if you can spawn them, unless you're starting the script before qb-core?

otherwise if its throwing an error when trying to craft, most likely a missing ingredient

**Comment by rpxyz** [comment]
No worries, I can see what has happened lol... the (readme) is outdated compared to what the list of items is on your GitHub, updated the list of items which is on here now and its working fine, lol sorry mate. Thanks :)
